pkg/datastore/projectstore.go

```diff
@@ -34,6 +36,19 @@ func (p *projectCollection) Factory() Factory {
 	}
 }
 
+func (p *projectCollection) MakeStoredFileNames(id string) []string {
+	return []string{fmt.Sprintf("%s.json", id)}
+}
+
+func (p *projectCollection) MakeUpdatableFileName(id string) (string, error) {
+	switch p.requestedBy {
+	case WebCommander:
+		return fmt.Sprintf("%s.json", id), nil
```

Comment:
Commander could be increased in the future so to be unified we should return `%s_api.json`, wdyt.

I mean we can think like our FileDB has some shards. Each shard is dedicated to one or some commanders.
So we have `piped`, `api`, `ops` shard at this time.

Reply:
Got your point, it's better indeed 👍

Reply:
In such case, I wonder which shard should be used in the case of `CommandStore`, we have 2 commanders: piped and ops (the orphanCommandCleaner in ops use this store). Looks like it's supposed to be separated into 2 shards: piped and ops but both commanders try to update the same field `command.Status` 🤔 
https://github.com/pipe-cd/pipecd/pull/3272/files#diff-c4043e761b81d90d0a6ef97287112ad4d0b20952d465605cba3510c3a40bf977R45